test/services/teams/helper/helper.user.js

```diff
@@ -0,0 +1,85 @@
+const rolesModel = require('../../../../src/services/role/model.js');
+const { userModel } = require('../../../../src/services/user/model.js');
+const accountModel = require('../../../../src/services/account/model.js');
+//const app = require(SRC + 'app');
+const { BadRequest } = require('feathers-errors');
+const { ObjectId } = require('mongoose').Types;
+const app = require('../../../../src/app');
+const {warn} = require('../../../../src/logger/index');
+
+const PASSWORD = process.env.TEST_PW ? process.env.TEST_PW.trim() : warn('process.env.TEST_PW is not defined');
```

Comment:
I still don't like this. The more configuration the tests need, the more annoying it will be for new devs. As all tests are going against the local test db (and we have passwords and hashes all over the place), it is unnecessary to use environment variables.

Reply:
1) I don't like hashes in code. A random/plain text password is ok. 
2) For test i want to create the account over the model and not over the services. 
=> If we have a workflow to create the hash and the account, i am happy if we remove the env's. 
But for this PR is it overhead. Let solved it later together.